library/init/meta/mk_has_reflect_instance.lean

```diff
@@ -56,6 +56,9 @@ private meta def for_each_has_reflect_goal : name → name → list (list name)
   solve1 (has_reflect_case I_name F_name ns),
   for_each_has_reflect_goal I_name F_name nss
 
+/-- Solves a goal of the form `has_reflect α` where α is an inductive type.
+    Needs to synthesize a `reflected` instance for each inductive parameter type of α
+    and for each constructor parameter of α. -/
```

Comment:
Oh, I see now I misread your comment. I will add comments to the implementation later.

Reply:
Updated.